source/_posts/2015-09-28-the-folly-of-fatherhood.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "The Folly of Fatherhood"
+date: 2015-09-28 21:21:48 -0500
+comments: true
+categories: 
+---
+Upon reflection, I think I have observed something about fatherhood. Or at
+least _my_ fatherhood (the one I am doing or having right now, not the one my
+dad did or had with, on, at or to me—the prepositions, here, are unclear to
+me). It is this: I am fairly certain I would _destroy myself_ saving one my my
+children. In fact, I’m pretty sure I’d destroy myself _failing_ such an
+attempt. Not infrequently, I think of a specific incident. (This may get a bit
+dark for some readers. There is talk of death and dying both my own and my
```

Comment:
Nice catch. Not semicolon (since "both my own and my children's" isn't a complete sentence), but maybe a colon. Or an em-dash.